notebooks/406-3D-pose-estimation-webcam/README.md

```diff
@@ -0,0 +1,21 @@
+# 3D Human Pose Estimation with OpenVINO 
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/spencergotowork/openvino_notebooks.git/master?labpath=notebooks%2F224-human-pose-estimation-3d%2F)
+*Binder is a free service where the webcam will not work, and performance on the video will not be good. For best performance, we recommend installing the notebooks locally.*
+
+![pose estimation_webgl](https://user-images.githubusercontent.com/42672437/183292131-576cc05a-a724-472c-8dc9-f6bc092190bf.gif)
+![pose estimation_opencv](https://user-images.githubusercontent.com/42672437/183285240-4ac00639-ceba-4b65-a783-be66a372ac8e.gif)
```

Comment:
Could you help to add some comments to illuminate the difference between these 2 notebooks ?